apps/web/src/components/layout/components/NotificationCenterWidget.tsx

```diff
@@ -14,7 +14,7 @@ export function NotificationCenterWidget({ user }: { user: IUserEntity | undefin
         backendUrl={API_ROOT}
         socketUrl={WS_URL}
         subscriberId={user?._id as string}
-        applicationIdentifier={environment?.identifier as string}
+        applicationIdentifier={process.env.REACT_APP_NOVU_APP_ID || (environment?.identifier as string)}
```

Comment:
```suggestion
        applicationIdentifier={APP_ID || (environment?.identifier as string)}
```
And add the import in line 5:
```
import { API_ROOT, APP_ID, WS_URL } from '../../../config';